src/sif/dynamiccost.cc

```diff
@@ -354,6 +358,11 @@ void DynamicCost::set_use_living_streets(float use_living_streets) {
              2.f * (1.f - use_living_streets) * (1.f - kMinLivingStreetFactor));
 }
 
+void DynamicCost::set_use_lit(float use_lit) {
+  unlit_factor_ =
+      use_lit < 0.5f ? kMinLitFactor + 2.f * use_lit : ((kMinLitFactor - 5.f) + 12.f * use_lit);
```

Comment:
> but I just can't see it happening in the code changes.

OK, I'm just blind (again).. There was a tiny bug there @chrstnbwnkl which caused the default to be a `unlit_factor_` of 2 (as it must've been, not sure why I didn't see it..).